PhysicsTools/PatAlgos/python/slimming/MicroEventContent_cff.py

```diff
@@ -144,6 +144,10 @@
                                         'keep L1GtTriggerMenuLite_l1GtTriggerMenuLite__*'
                                       ]
 
+from Configuration.Eras.Modifier_strips_vfp30_2016_cff import strips_vfp30_2016
+strips_vfp30_2016.toModify(MicroEventContentMC, outputCommands = MicroEventContentMC.outputCommands + [
+    'keep *_*_SimulatedAPVDynamicGain_*'
```

Comment:
please use specific module instance names

Reply:
BTW, is this collection always produced now, or just with this strips_vfp30_2016 era?
It may be easier to just add to the default ```keep``` list to keep this config simpler.

Reply:
I added the module instance name.
>  is this collection always produced now, or just with this strips_vfp30_2016 era?

it is produced always, but it is not relevant in any other era.

>  It may be easier to just add to the default keep list to keep this config simpler.

Perhaps so, but as this eventually needs to be backported to 10.6.X I am not sure one can change the event content of the production UL release **also** for the years already processed. Maybe @fabiocos can clarify the stance on this.